### PR TITLE
Updated schema and structure

### PR DESCRIPTION
**Pull request by sarken**

### CODE REVIEW

File: db/schema.rb [hunk 1]
```diff
@@ -11,7 +11,7 @@
 #
 # It's strongly recommended to check this file into your version control system.
 
-ActiveRecord::Schema.define(:version => 20010101010101) do
+ActiveRecord::Schema.define(:version => 20150901132832) do
```
Comment (zz9pzza) [inline db/schema.rb]: This bit needs to be reset back

Comment (elzj) [inline db/schema.rb]: @zz9pzza Why? It should always be up to date.

Comment (zz9pzza) [inline db/schema.rb]: In the past when we have done that it has meant that it has ignored migrations before that date.

Comment (elzj) [inline db/schema.rb]: But that shouldn't be - I've run a lot of migrations, and I haven't seen that happen. The schema table keeps a list of which ones have run.

**Pull request merged**